Example/Optik/ViewController.swift

```diff
@@ -82,4 +82,20 @@ extension ViewController: ImageViewerDelegate {
         currentLocalImageIndex = index
     }
     
+    func didTouchLike(button: UIButton, at index: Int) {
```

Comment:
`didTouchLike` -> `actionbuttonTapped`

Please get rid of the word "like". Use the search feature to make sure you didn't forget any!